maestro-domain/pom.xml

```diff
@@ -23,7 +23,7 @@
     <parent>
         <artifactId>maestro</artifactId>
         <groupId>bio.overture</groupId>
-        <version>${revision}${sha1}${changelist}</version>
+        <version>3.8.0</version>
```

Comment:
You don't need to do this in Maestro, you change the .mvn/conf file for the version

Reply:
reverted